packages/ember-metal-views/lib/htmlbars-renderer.js

```diff
@@ -84,6 +84,7 @@ Renderer.prototype.dispatchLifecycleHooks =
         case 'didUpdate': this.didUpdate(hook.view); break;
       }
 
+      ownerView._dispatching = 'didRender';
```

Comment:
just double checking, `_dispatching` is already defined in the constructor of ownerView ?